src/fetch.js

```diff
@@ -25,6 +25,26 @@ const fetchAPI = {
       .then(response => fetchAPI.handleFetchError(response))
       .then(response => response.json())
       .catch(error => console.log(error));
+  },
+
+  postNewTrip: (trip) => {
+    return fetch("http://localhost:3001/api/v1/trips", {
+      method: 'POST',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify({
+        'id': trip.id,
+        'userID': trip.userID,
+        'destinationID': trip.destinationID,
+        'travelers': trip.travelers,
+        'date': trip.date,
+        'duration': trip.duration,
+        'status': trip.status,
+        'suggestedActivities': trip.suggestedActivities
+      })
+    })
+    .then(response => fetchAPI.handleFetchError(response))
+    .then(response => response.json())
```

Comment:
You can make handleFetchError return response.json() instead of just response and basically combine these lines. I don't think one way is better than the other but just something I wanted to point out.